### PR TITLE
Correcting maven-dependency-plugin issue #5272

### PR DESCRIPTION
**Pull request by fliot**
This pull request allow successfull maven build, without "<mdep.analyze.skip>true</mdep.analyze.skip>" profile settings.

It correspond to the issue #5272 

Regards.

### CODE REVIEW

**Comment by skabashnyuk** [comment]
ci-build

**Comment by skabashnyuk** [comment]
ci-build

File: wsmaster/che-core-api-auth/pom.xml [hunk 1]
```diff
@@ -120,6 +116,11 @@
         </dependency>
         <dependency>
             <groupId>org.everrest</groupId>
+            <artifactId>everrest-core</artifactId>
```
Comment (skabashnyuk) [inline wsmaster/che-core-api-auth/pom.xml:119]: 11:02:07 /home/codenvy/workspace/che-pullrequests-build/wsmaster/che-core-api-auth/src/main/java/org/eclipse/che/security/oauth/OAuthAuthenticator.java:234: error: cannot access JsonValue
11:02:07             return JsonHelper.fromJson(urlInputStream, userClass, null);
11:02:07                              ^
11:02:07   class file for org.everrest.core.impl.provider.json.JsonValue not found
11:02:07 1 error
11:02:08 [JENKINS] Archiving disabled

**Review by skabashnyuk — CHANGES_REQUESTED** [review]
The source code is not compiling.

**Comment by mkuznyetsov** [comment]
Correct scopes have been set for all dependencies in this PR
https://github.com/eclipse/che/pull/5416

**Pull request closed without merging**